### PR TITLE
Add s3 CDN and conditionally set clippy cdn variable based on the window property

### LINKED ISSUE #319: CDN's for clippy are down
Can't reach the CDN's for clippy over at 

https://gitcdn.xyz/repo/pi0/clippyjs/master/assets/agents/Clippy/agent.js

and

https://gitcdn.xyz/repo/pi0/clippyjs/master/assets/agents/Clippy/sounds-mp3.js

### PR DESCRIPTION
**Pull request by Mrpanquecas**
fix(clippy): add s3 CDN and conditionally set clippy cdn variable based on the window property

fix #319

### CODE REVIEW

File: packages/clippy/src/ClippyProvider.jsx [hunk 1]
```diff
@@ -6,6 +6,8 @@ import clippyStyle from './style';
 import AGENTS from './agents';
 import ClippyContext from './ClippyContext';
 
+if (window) window.CLIPPY_CDN = '//s3.amazonaws.com/clippy.js/Agents/';
+
```
Comment (ggdaltoso) [inline packages/clippy/src/ClippyProvider.jsx:10]:
```suggestion
if (typeof window !== 'undefined') window.CLIPPY_CDN = '//s3.amazonaws.com/clippy.js/Agents/';

```

On server side, `window` will be `undefined` and will make this a false positive. We need to ensure that window isn't `undefined`

**Review by ggdaltoso — CHANGES_REQUESTED** [review]
Thanks for your first contribution, @Mrpanquecas !!

I've just made a simple comment to make our condition ever better

File: packages/clippy/src/ClippyProvider.jsx (hunk 1 above)
Comment (Mrpanquecas) [reply packages/clippy/src/ClippyProvider.jsx:10]: if window is undefined javascript will treat it as a falsy value but I do agree the condition should be more explicit specially for readability. I'll make an update tonight

**Review by ggdaltoso — APPROVED** [review]
LGTM!!🚀 

Thanks again, @Mrpanquecas!

**Pull request merged**